README.md

```diff
@@ -1,3 +1,7 @@
+# Warning: this repository has been moved
+
+It now lives in the [kiwicom/orbit](https://github.com/kiwicom/orbit) monorepo.
+
```

Comment:
Could you also add a horizontal rule below so it stands out a bit more?

Reply:
Added 🙂